resources/views/fly/entrypoint.blade.php

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# Run user scripts, if they exist
+for f in /var/www/html/.fly/scripts/*.sh; do
+    # Bail out this loop if any script exits with non-zero status code
+    bash "$f" || break
```

Comment:
I realize that you are just copying the file from flyctl, but the above seems wrong to me.  Run a list of scripts, but if there is any failure stop running those scripts and proceed on?

I'd suggest adding `-e` to the end of the line at the top which tells the script interpreter to exit immediately on first failure and drop the `|| break`.

Reply:
@rubys  Hmmm. This makes sense! Exit the entrypoint if any of the provided scripts fail. 

One issue with this is on exit, it ultimately stops running the supervisor found in line 14, and therefore blocks the starting of the Laravel server.